docs/development/issue_labeling.md

```diff
@@ -0,0 +1,228 @@
+# Issue labeling
+
+We try to keep issues well-classified by labeling them appropriately.
+Only members/collaborators with the required rights can apply labels.
+
+## Basic knowledge about Renovate
+
+You should know about platforms, package managers, datasources and versioning to label issues effectively.
+
+- To learn about platforms, read the [Renovate docs on Platforms](https://docs.renovatebot.com/modules/platform/).
+- To learn about managers, read the [Renovate docs on Managers](https://docs.renovatebot.com/modules/manager/).
+- To learn about datasources, read the [Renovate docs on Datasources](https://docs.renovatebot.com/modules/datasource/).
+- To learn more about versioning, read the [Renovate docs on Versioning](https://docs.renovatebot.com/modules/versioning/).
+
+## Label categories
+
+### Severity
+
+<details>
+    <summary>Severity</summary>
+
+    breaking
+    major
+
+</details>
+
+QUESTION: When/who should use these severity labels?
+
+### Priority
+
+<details>
+    <summary>Priority</summary>
+
+    pri0-blocker
+    pri1-critical
+    pri2-important
+    pri3-normal
+    pri4-low
+    wontfix
+
+</details>
+
+Use these to assign a priority level to an issue.
+
+QUESTION: Who gets to decide the priority level? @rarkins and maintainers, or can a triaging member select a priority level?
+
+QUESTION: Maybe we should copy paste the priority descriptions from the labels into this doc?
```

Comment:
This is an important field so we could start by copying but best if we expand it.

Reply:
I've added descriptions to those which were missing